recipe/meta.yaml

```diff
@@ -42,17 +40,17 @@ outputs:
       run:
         - python
         - deprecated >=1.2.13
+        - numba >=0.53
         - numpy >=1.21.0,<1.25
-        - pandas >=1.1.0,<1.6.0
-        - numba >=0.55
+        - pandas >=1.1.0,<2.0.0
         - scikit-learn >=0.24.0,<1.3.0
-        - statsmodels >=0.12.1
         - scipy >=1.2.0,<2.0.0
       run_constrained:
         - esig ==0.9.7
         - filterpy >=1.4.5
```

Comment:
Do we need a python selector here? [py<311]
https://github.com/sktime/sktime/blob/v0.17.0/pyproject.toml#L62

Reply:
All available packages have python 3.11 support, except pystan ==2.19.1.1
```
        - gluonts >=0.9.0  # it's noarch package
        - hmmlearn >=0.2.7  # It's not available on defaults
        - kotsu >=0.3.1  # it's not available on defaults
        - matplotlib-base >=3.3.2
        - pmdarima >=1.8.0,!=1.8.1,<3.0.0
        - prophet >=1.1
        - pykalman >=0.9.5  # It's not available on defaults
        - pyod >=0.8.0
        - pystan ==2.19.1.1  # it has only py38 and py39 support
        - scikit-posthocs >=0.6.5  # It's not available on defaults
        - seaborn >=0.11.0
        - statsforecast >=0.5.2  # It's not available on defaults
        - statsmodels >=0.12.1
        - stumpy >=1.5.1
        - tbats >=1.1.0
        - tsfresh >=0.17.0  # it's noarch package
        - tslearn >=0.5.2  # it's not available on defaults
```